src/CommunityToolkit.Maui.Core/Essentials/Badge/BadgeImplementation.macios.cs

```diff
@@ -23,7 +23,7 @@ public void SetCount(uint count)
 			{
 				if (error is not null)
 				{
-					Trace.WriteLine($"Error setting the Badge Count: {error.Description}");
+					Trace.TraceError($"Error setting the Badge Count: {error.Description}");
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.